extensions/wikia/CommunityPage/styles/CommunityPage.scss

```diff
@@ -27,8 +27,16 @@
 }
 
 .CommunityPageMainContent {
+	@include flex-grow(1);
+
 	box-sizing: inherit;
 	padding-top: 30px;
+	> * {
+		// 3 biggest card size with margins
+		max-width: 3 * $community-page-card-module-width-regular + 3 * $content-spacing-regular;
```

Comment:
Are you sure this is supposed to be 3 * $content-spacing-regular and not 2 times?